woocommerce/woocommerce
# Add/35076

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/35076






### How to test the changes in this Pull Request:



1. Go to `Products -> Add New (MVP)` page
2. The category select, attributes select attribute terms select and all form controls that uses the experimental select-control should perform as expected.



### Other information:




### FOR PR REVIEWER ONLY:
